packages/modular-scripts/src/cli.ts

```diff
@@ -118,6 +128,38 @@ function run() {
   }
 }
 
+interface WorkspaceMeta {
+  location: string;
+  workspaceDependencies: string[];
+  mismatchedWorkspaceDependencies: string[];
+}
+
+function listPackages(typeArg: string | void) {
```

Comment:
I don't think that `| void` is correct (although I can see it used below - I don't think that's right though). `void` is usually used for functions that don't return anything. Did you mean this?

```suggestion
function listPackages(typeArg?: string) {
```